src/model/element/interaction-type/binding.js

```diff
@@ -39,7 +47,7 @@ class Binding extends InteractionType {
   }
 
   toString(){
-    return super.toString('binds');
+    return super.toString('binds with');
```

Comment:
"A binds B" sounds better than "A binds with B", which has also a different meaning in English, i.e. and e.g: "A binds with B" usually means B is a tool or substance to stick A to smth. That's why I removed "with" before.

Reply:
The doc says "A binds to B".  I don't think we should use "to", since that implies direction.  I think "A binds with B" sounds more undirected than "A binds B".

@jvwong, what do you think?